Runtime/Triangulator.cs

```diff
@@ -231,6 +231,8 @@ public class TriangulationSettings
         public class InputData
         {
             public NativeArray<float2> Positions { get; set; }
+            
+            public NativeArray<bool> IgnorePositions { get; set; }
```

Comment:
I'm not native english but imho `IgnoredPositions` is better than `IgnorePositions`, what do you think?

Reply:
I disagree, because it's not an array of positions that are ignored, it's an array of whether or not to ignore the positions.

If I see a value called `IgnoredPositions`, I expect it to be the same type of data as `Positions`, since the name is the same as `Positions` just with an adjective added to describe those positions. I expect an array of `float2` that contains the coordinates of positions to be ignored.

`IgnorePositions` has more clarity imho. It implies the question, "do we ignore these positions?" which can cleanly be answered with yes/true or no/false. Also, "ignore" is a present-tense verb (whereas "ignored" is an adjective or maybe a past-tense verb) and in general it's good practice to name booleans with a present-tense verb.

If you still think `Ignored` is better I'm ok with that too, but this was my reasoning for choosing `Ignore` :)